command/untag.go

```diff
@@ -0,0 +1,90 @@
+package command
+
+import (
+	"flag"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+)
+
+type UnTagCommand struct {
+	Meta
+}
+
+func (c *UnTagCommand) Run(args []string) int {
+	var (
+		dryRun           bool
+		instanceIDString string
+		instanceIDs      []*string
+		tagString        string
+		tags             []*ec2.Tag
+	)
+
+	var (
+		arguments []string
+	)
+
+	svc := ec2.New(session.New(), &aws.Config{})
+
+	flags := flag.NewFlagSet("dtan4", flag.ContinueOnError)
+	flags.Usage = func() {}
+
+	flags.BoolVar(&dryRun, "dry-run", false, "Dry run (default: false)")
+	flags.StringVar(&instanceIDString, "instance", "", "Instance Ids")
+	flags.StringVar(&tagString, "tags", "", "KEY=VALUE tags")
```

Comment:
Does it need to take `value` in addition to `key`? I think only the tag name is enough to remove a certain tag.